woocommerce/woocommerce-gutenberg-products-block
# Shopper → Mini Cart → Can see the icon with a badge

Fixes #5828

### Notes

This PR adds e2e tests to verify that a shopper can see the Mini Cart icon with the amount, icon, and quantity badge:
- All shopper flows related to Mini Cart will happen on the Mini Cart Block Page.
- `All Products` block was added to the Mini Cart block page.
-  `ensureSidebarOpened` was taken from the latest `trunk` of `@wordpress/e2e-test-utils` to fix the failed test when the sidebar is closed. I did that because we're using [the older version of `@wordpress/scripts`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/5349).

### Testing

#### Automated Tests

### Manual Testing

1. Check out this PR.
2. Start Docker Desktop.
3. Run `npm run wp-env destroy && npm run wp-env start` to start a fresh e2e environment.
4. Run `npm run test:e2e -- tests/e2e/specs/shopper/mini-cart-icon.test.js` to run only this test case.
5. Run `npm run test:e2e` to run the whole test suite.
